include/config/overworld.h

```diff
@@ -107,5 +107,7 @@
 
 // Pokémon Center
 #define OW_IGNORE_EGGS_ON_HEAL     GEN_LATEST               // In Gen 4+, the nurse in the Pokémon Center does not heal Eggs on healing machine.
```

Comment:
I would indent `GEN_LATEST` to align with the new configs right below it

Reply:
[`a8e9724` (#5448)](https://github.com/rh-hideout/pokeemerald-expansion/pull/5448/commits/a8e972429c4b86be20ed72da04107652f6cf4c88)